apps/emqx/test/emqx_client_SUITE.erl

```diff
@@ -43,6 +43,8 @@
     <<"TopicA/#">>
 ]).
 
+-define(WAIT(EXPR, ATTEMPTS), wait(fun() -> EXPR end, ATTEMPTS)).
```

Comment:
Nit: https://github.com/kafka4beam/snabbkaffe/blob/master/include/test_macros.hrl#L180 ? (included in snabbkaffe.hrl)